templates/base.j2

```diff
@@ -14,7 +14,7 @@
 </head>
 <body>
 
-  {{ iconsSvg }}
```

Comment:
So... is this a terrible idea from a security standpoint? It's required to make this feature work with recent versions of Nunjucks.